### ISSUE TITLE
Seeing multiple lines output for new version warning

### ISSUE THREAD

**Issue by justincampbell** [hexpm/hex#223]
Hex: 0.11.3
Elixir: 1.2.3
OTP: 18.3
Phoenix:  v1.1.4

```
mix phoenix.new  hex_test
cd hex_test
mix deps.get
iex -S mix
```

Output:

![](http://cl.ly/1w0S0I0d1x23/Screen%20Shot%202016-04-11%20at%203.06.48%20PM.png)

Something else is probably at fault here for calling the relevant Hex function multiple times, but I'm not sure how to track it down. This doesn't happen for a plain mix project, so probably from Phoenix.

https://twitter.com/wojtekmach/status/719609806785523712

**Comment by justincampbell** [comment]
Based on the commit message, this is probably resolved by https://github.com/hexpm/hex/commit/4c1122cc27d892d40ea97ee7fe2ca8dba5e736f8. Feel free to reopen if I'm incorrect. Thanks!

**Comment by ericmj** [comment]
It's there to *really* make sure you know there's an update ;).

You are indeed correct, it was fixed in 4c1122c and the fix was released with v0.11.5.
